suite-native/app/package.json

```diff
@@ -73,44 +73,43 @@
         "@trezor/styles": "workspace:*",
         "@trezor/theme": "workspace:*",
         "buffer": "^6.0.3",
-        "expo": "50.0.6",
-        "expo-av": "13.10.5",
-        "expo-barcode-scanner": "12.9.2",
-        "expo-build-properties": "^0.11.1",
-        "expo-clipboard": "5.0.1",
-        "expo-constants": "15.4.5",
-        "expo-dev-client": "^3.3.7",
-        "expo-haptics": "12.8.1",
-        "expo-image": "1.10.6",
-        "expo-linear-gradient": "12.7.1",
-        "expo-localization": "14.8.3",
-        "expo-navigation-bar": "2.8.1",
-        "expo-secure-store": "12.8.1",
-        "expo-splash-screen": "0.26.4",
-        "expo-status-bar": "~1.11.1",
-        "expo-system-ui": "2.9.3",
-        "lottie-react-native": "6.6.0",
+        "expo": "^51.0.9",
```

Comment:
@Nodonisko Why did you switch from the exact version to the carret range everywhere?

Reply:
default behavior of yarn up command, don't want to fight it

Reply:
It could be configured so I will update it to exact versions again

Reply:
I belive there is no real difference for us, I was just curious...